extensions/amp-consent/amp-consent.md

```diff
@@ -278,6 +278,20 @@ AMP will check client cache and server in parallel to find the previous consent
 }
 ```
 
+#### consentMetadata
```

Comment:
Given people are actively making changes. Should we 
1. update the doc after the change hit prod
or
2. update the doc together with complete consentMetadata support.

Reply:
Ahh probably the latter. Removing these changes for now.

Reply:
SG